documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -146,14 +150,76 @@ Example CPU images:
+  - The WebLogic generic image is supported for development and production deployment
+    of WebLogic configurations using Docker.
+  - Contains the same binaries as those installed by the WebLogic generic installer.
+- `.../weblogic...:...slim...`:
+  - The WebLogic slim image is supported for development and production deployment
+    of WebLogic configurations using Docker.
+  - In order to reduce image size,
+    it contains a subset of the binaries included in the WebLogic generic image:
+    - The WebLogic Console, WebLogic examples, WebLogic clients, Maven plug-ins,
+      and Java DB have been removed.
+    - All binaries that remain included are
+      the same as those in the WebLogic generic image.
+  - If there are requirements to monitor the WebLogic configuration,
+    they should be addressed using Prometheus and Grafana, or other alternatives.
+- `.../weblogic...:...dev...`:
+  - The WebLogic developer image is supported for development 
+    of WebLogic applications in Docker containers.
+  - In order to reduce image size, it contains a subset
+    of the binaries included in the WebLogic generic image:
+    - WebLogic examples and Console help files have been removed.
+    - All binaries that remain included are the same as those in the WebLogic generic image.
+  - This image type is primarily intended to provide a Docker image
+    that is consistent with the WebLogic "quick installers" intended for development only.
+    Production WebLogic domains should use the WebLogic generic or WebLogic slim images.
+- `.../fmw-infrastructure...:...`:
+  - The Fusion Middleware (FMW) Infrastructure image is supported for
+    development and production deployment of FMW configurations using Docker.
+  - It contains the same binaries as those installed by the WebLogic generic installer
+    and adds Fusion Middleware Control and Java Required Files (JRF)
+
+Notes about "undated" OCR images with name tags that do _not_ include an embedded date stamp:
+- They represent a GA version. 
+  _Therefore they may be used in samples and development, but are
+  not recommended for production use._
+- Unlike images with an embedded datastamp,
+  which represent a specific version,
+  undated images are periodically updated to 
+  the latest available versions of their GA equivalents.
+  _Therefore they change over time in the repository
+  even though their name and tag remain the same._
+- Examples of undated images include
+  `registry.oracle.com/middleware/weblogic:TAG` images
+  where `TAG` is one of `12.2.1.3`, `12.2.1.4`, `14.1.1.0-11`, or `14.1.1.0-8`.
+  These are created with the generic installer, 
+  Oracle Linux 7-slim, and JDK8 
+  except for `14.1.1.0-11` (which has JDK11).
+- The tag for an undated image may not 
+  specify its WebLogic installer type, 
+  in which case you can assume it is `generic`.
+  Otherwise, such images may have
+  a tag that includes the string `slim` or `dev`
+  string, in which case they have the 
```

Comment:
includes the string `slim` or `dev` string, -> remove one instance of "string"

Reply:
no longer applicable (and text now moved to image doc)